assets/js/account/account-manager.tpl.liquid

```diff
@@ -0,0 +1,65 @@
+<div ng-if="$ctrl.loader.isLoading" class="loader">
```

Comment:
We are using a different loader now, pls check 'assets/js/account/b4/account-manager.tpl.liquid' file